cornice_swagger/swagger.py

```diff
@@ -147,7 +147,7 @@ def _build_paths(self, ignore=['head', 'options'], default_tags=None,
 
         return paths, tags
 
-    def _extract_path_from_service(self, service):
+    def _extract_path_from_service(self, service, **kwargs):
```

Comment:
Hmm. Be careful with this kind of pattern (`**kwargs` that gets shared across lots of methods, each of which pays attention to only a few). If you typo a parameter name, it will get silently ignored instead of being flagged as an error. The alternative is to explicitly list all the keyword arguments that any of these extractor methods supports, which is very tedious but at least predictable. Of course, that problem isn't specific to this PR, so I guess it's fine for now.